tmt/base.py

```diff
@@ -2228,14 +2303,14 @@ def stories(
 
         # Build the list, convert to objects, sort and filter
         stories = [
-            Story(node=story) for story
+            Story(node=story, logger=self._logger.descend()) for story
```

Comment:
Good question. I would say no, hence `.descend()`, but it's up to us to decide. IIRC, this matches the current output, where each story is indented by this much, which `.descend()` achieves. As I see the semantics of `logger` not just here, but also in `tests()` and `plans()`, the logger is given to these methods for their logging, and to serve as a base for objects these methods return. By using the same for both, the method would log on the same level as each test, which doesn't seem right to me - or, rather, tests/plans/stories logging on the same level as `Tree` that *owns* them is what I think of as "not desirable".

Reply:
Sorry, probably a misunderstanding? I meant using `logger` instead of `self._logger` for descending:

```suggestion
            Story(node=story, logger=logger.descend()) for story
```

To be consistent with the way it's done for `tests()` and `plans()` above. Basically do not ignore `stories(..., logger)`.

Reply:
Oh, right. That's even wrong, `self._logger` shouldn't be used, `logger` is the only right answer there. Fixed in 38c3be9b8b0789fae775146894cc614438101bb0

Reply:
Good.